packages/compass-saved-aggregations-queries/src/stores/delete-item.ts

```diff
@@ -3,6 +3,9 @@ import { FavoriteQueryStorage } from '@mongodb-js/compass-query-history';
 import { PipelineStorage } from '@mongodb-js/compass-aggregations';
 import type { ThunkAction } from 'redux-thunk';
 import type { RootState } from '.';
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
```

Comment:
do we need to `npm install @mongodb-js/compass-logging` in this package?

Reply:
Yes, fixed, thanks